src/components/home/Home.tsx

```diff
@@ -1,10 +1,60 @@
 import { ReactElement } from "react";
+import GitHub from "../icons/gith-hub";
+import { Button } from "../ui/button";
+import { Link } from "react-router-dom";
+import * as qs from 'qs';
+import { generateCSRFToken } from "../../lib/security";
+import { useStoreInContext } from "../../lib/zustand";
+import { ProviderName } from "../../requests/integrations/oauth2/github";
+
+// TODO[P3]: Support for other git providers
+const ACCEPTED_GIT_PROVIDERS = new Set(["github"]);
```

Comment:
This is totally fine, but in another location, [src/requests/integrations/oauth2/github.ts](https://github.com/sweepdotdev/sweep-client/pull/2/files#diff-8b340a3515447185167d51f7febd25e338d200ab1d4a309b08da6a7e3b2fa4b4), it's a union. 

Unless there's a specific reason, probably best to standardize. You could probably even export this variable for global use.